frontend/README.md

```diff
@@ -0,0 +1,14 @@
+# URL-S Frontend
+
+Instructions for configuring and starting the URL-S frontend app, Vue.js, and its dependencies.
+
+It is recommended that you install and start the Backend applications before using the Frontend. See the README in the Backend folder.
+
+## Local Setup
+**`Step 1`**. Open a terminal (option to do this in MS Code). Change directory to the frontend/url-shortener folder
+  
+**`Step 2`**. From the /url-shortener folder, run `npm install` in the terminal to install required dependencies.
+ 
+**`Step 3`**. From the /url-shortener folder, run `npm run dev` in the terminal to start the app.
+
+**`Step 4`**. Open a browser and visit the app dev site at http://localhost:5173
```

Comment:
We should add here on testing the app once it is installed. Like how to check the values in the database to confirm the flow

Reply:
@veenupunyani I cannot do this quickly. I will make a new ticket to add testing steps to the READMEs